apps/files/decorators.py

```diff
@@ -18,21 +18,22 @@
 
 
 def allowed(request, file):
-    allowed = acl.check_addons_reviewer(request)
-    if not allowed:
-        try:
-            addon = file.version.addon
-        except ObjectDoesNotExist:
-            raise http.Http404
-
+    try:
+        addon = file.version.addon
+    except ObjectDoesNotExist:
+        raise http.Http404
```

Comment:
Yeah... didn't want to potentially raise new 500s in production (even if highly unlikely).